lib/MultiColumnList/MCLRenderer.js

```diff
@@ -1335,7 +1373,14 @@ class MCLRenderer extends React.Component {
         headerInner = this.getMappedColumnName(header);
       }
 
-      const shouldMeasure = (!columnWidthsProp[header] && this.headerWidths.request(header) === null);
+      let shouldMeasure = false;
+      const headerMeasured = this.headerWidths.request(header);
+      if (modColumns.includes(header) && headerMeasured === null) {
```

Comment:
I think it can be simplified a bit.
```suggestion
     const shouldMeasure = modColumns.includes(header) && headerMeasured === null || !columnWidthsProp[header] && this.headerWidths.request(header) === null;
```

As a further improvement IMO would be storing `modColumns.includes(header) && headerMeasured === null` and `!columnWidthsProp[header] && this.headerWidths.request(header) === null` into its own variables for a better readability.